gcn_classic_to_json/notices/GECAM_FLT/__init__.py

```diff
@@ -0,0 +1,39 @@
+from ... import utils
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[27:39]  # Unused. According to docs: '32 bytes for the future'
+    bin[
+        11
+    ]  # Temporarily Unused. Should have a int->class_type mapping but incomplete documentation
+    bin[
+        18
+    ]  # Temporarily Unused. Should have a bit->flag mapping but incomplete documenation
+    bin[21]  # Temporarily Unused. How is it different from burst_dur?
+    bin[22:24]  # Unused. Equivalent to rate_energy_range
+    bin[
+        26
+    ]  # Temporarily Unused. Presumably which detectors activated but incomplete documenation
+    return {
+        "mission": "GECAM",
+        "id": [bin[4]],
+        "messenger": "EM",
+        "mission_type": chr(bin[19] + 64),  # mission type is A or B
+        # There is an error here; You are supposed to divide by 8640000 to get the correct value
+        # But it only seems to work if you divide by 864000000; I'm assuming it's some error in encoding the packets
+        "trigger_time": utils.datetime_to_iso8601(bin[5], bin[6] * 1e-2),
```

Comment:
What value do you get for the datetime if you remove the `* 1e-2`?

Reply:
With the `*1e-2` its -> "trigger_time": "2024-05-17T13:01:22.550Z"
Without the `*1e-2` its -> "trigger_time": "2024-07-10T06:17:35.040Z",

Reply:
How do you know which one is correct?

Reply:
When I first started GECAM script for text conversion was the first week of July and so the trigger time at that time was a few days in the future. Additionally, with the factor of `1e-2` it matches the text notices. I know we shouldn't treat the text notices as truth, but the month of May seems to line up with the trigger time for the other notice types as well. I'm assuming that the repo was created around this and the latest binaries were taken.